### ISSUE TITLE
Faking entities

### ISSUE THREAD

**Issue by pluxain** [deerawan/vscode-faker#16]
Hi,

This is a question, not an issue.

Is it possible somehow to fake more complex data structure, using fakers provided here. Like :
faker person -> 
{ 
  name: faker name,
  address: faker address,
  phone: faker phone
}

**Comment by deerawan** [comment]
@pluxain do you mean that when typing `faker person`, it will generate that object? If yes, it is not possible atm 😄

**Comment by pluxain** [comment]
Yes this is what I meant. I asked in case it was be possible somehow to set this kind of faker ourselves in a file or something. Anyway, your plugin already saves me loads of time, so thank you for that

**Comment by arash-bizcover** [comment]
Object creation would be extremely useful, similar to this original helper **faker.helpers.createTransaction()**

https://github.com/Marak/faker.js/blob/master/lib/helpers.js#L356-L374
